apps/chromium-browser/umbrel-app.yml

```
@@ -0,0 +1,19 @@
+manifestVersion: 1
+id: "chromium-browser"
+name: "Chromium Browser"
+tagline: "Browse securely with LinuxServer.io's Chromium"
+description: "A full-featured Chromium browser with secure remote access. Perfect for browsing the web from any device, with built-in security and easy access through your Umbrel."
+version: "1.0.0"
+category: "networking"
+port: 30099
+developer: "LinuxServer.io"
+website: "https://hub.docker.com/r/linuxserver/chromium"
```

This is the original website:
```suggestion
website: "https://docs.linuxserver.io/images/docker-chromium"
```